features/prompt_memes_a/challenge_promptmeme_setup.feature

```diff
@@ -172,7 +172,7 @@ Feature: Prompt Meme Challenge
   Then I should see "Edit"
     And I should see "Delete"
   
-  Scenario: Edit individual prompt via signup show
+  Scenario: Edit individual prompt and get invalid update error
```

Comment:
We can name this scenario "Edit individual prompt".

Reply:
Done!